integration/hurl/tests_ok/secret/secret.sh

```diff
@@ -4,8 +4,8 @@ set -Eeuo pipefail
 rm -rf build/secret
 
 hurl --very-verbose \
-    --secret a=secret1 \
-    --secret b=secret2 \
+    --secrets-file tests_ok/secret/secrets0.env \
```

Comment:
This integration file should not be touched, we can make a new integration test specifi for `--secrets-file` in `tests_ok/secret/secrets_file.sh`